playground/mocks/idp/idx/introspect/data/identify.json

```diff
@@ -44,6 +44,10 @@
           ]
         }
       ]
+    },
+      "messages": {
```

Comment:
does the `identify` response actually have empty `messages` ? otherwise we shall handle the null-able in code.

Reply:
yeah, I tried this on Peter's branch and it has the empty array. They have a JIRA to fix it post LEA